phpBB/language/en/acp/forums.php

```diff
@@ -87,7 +87,7 @@
 	'FORUM_DELETE_EXPLAIN'				=> 'The form below will allow you to delete a forum. If the forum is postable you are able to decide where you want to put all topics (or forums) it contained.',
 	'FORUM_DELETED'						=> 'Forum successfully deleted.',
 	'FORUM_DESC'						=> 'Description',
-	'FORUM_DESC_EXPLAIN'				=> 'Any HTML markup entered here will be displayed as is.',
+	'FORUM_DESC_EXPLAIN'				=> 'Any HTML markup entered here will be displayed as is. If the selected forum type is category the description is not used.',
```

Comment:
should be?

"If the selected forum type is a category the description is not used"

Reply:
Done